src/shared/cloudformation/cloudformation.ts

```diff
@@ -44,7 +44,7 @@ export namespace CloudFormation {
         Runtime?: string
         MemorySize?: number
         Timeout?: number
-        Environment?: Environment
+        Environment?: unknown
```

Comment:
I don't see any reason why this should be unknown.

Reply:
We are treating this as a big blob of data and not validating it, `unknown` should be the correct type to shuffle it around